### PR TITLE
feat: update jest config

### PR DESCRIPTION
**Pull request by PavelKirvelTextkernel**
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. Use imperative, present tense in your commit description ("change", not "changed" or "changes") without uppercases or period (.) at the end.

# Checklist
- [ ] The implementation has been manually tested and complies with Textkernel [browser support guidelines](https://textkernel.com/browser-support/)
- [ ] The implementation complies with [accessibility](CONTIRBUTING.md#accessibility) standards.
- [ ] The component has a [displayName](CONTRIBUTING.md#display-names) defined.
- [ ] The component comes with a detailed [PropTypes](CONTRIBUTING.md#component-props) (and defaultProps) definition.
- [ ] Component PropTypes are sufficiently described / documented.
- [ ] There is [a story](CONTRIBUTING.md#component-showcases) in Storybook.

### CODE REVIEW

File: src/components/SelectComponents/Combobox/__tests__/Combobox.spec.tsx [hunk 1]
```diff
@@ -35,159 +40,227 @@ describe('Combobox', () => {
...
+                    noSuggestionsPlaceholder={noSuggestionsPlaceholder}
+                    onSelectionAdd={mockOnSelectionAdd}
+                    onInputValueChange={mockOnInputValueChange}
+                    onBlur={mockOnBlur}
+                    disabled
+                />
+            );
+            setFocusOnInput(viewContainer);
+
+            expect(viewContainer.querySelectorAll('input')).toHaveLength(0);
```
Comment (eszthoff) [inline src/components/SelectComponents/Combobox/__tests__/Combobox.spec.tsx:70]: s/a

File: src/components/SelectComponents/Combobox/__tests__/Combobox.spec.tsx [hunk 2]
```diff
@@ -35,159 +40,227 @@ describe('Combobox', () => {
...
+                    disabled
+                />
+            );
+            setFocusOnInput(viewContainer);
+            // expect(viewContainer.querySelectorAll('li')).toHaveLength(1);
 
-            expect(wrapper.find('li')).toHaveLength(1);
-            expect(wrapper.find('li').childAt(0).text()).toEqual(noSuggestionsPlaceholder);
+            // expect(wrapper.find('li')).toHaveLength(1);
+            // expect(wrapper.find('li').childAt(0).text()).toEqual(noSuggestionsPlaceholder);
```
Comment (eszthoff) [inline src/components/SelectComponents/Combobox/__tests__/Combobox.spec.tsx:89]: there are some commented out lines, but no actual test that will run in this test.

File: src/components/SelectComponents/Combobox/__tests__/Combobox.spec.tsx [hunk 3]
```diff
@@ -35,159 +40,227 @@ describe('Combobox', () => {
...
         });
         it('should render all suggestions from the list', () => {
-            setFocusOnInput();
+            setFocusOnInput(viewContainer);
 
-            expect(wrapper.find('li')).toHaveLength(suggestions.length);
+            expect(viewContainer.querySelectorAll('li')).toHaveLength(suggestions.length);
```
Comment (eszthoff) [inline src/components/SelectComponents/Combobox/__tests__/Combobox.spec.tsx:94]: s/a

File: src/components/SelectComponents/Combobox/__tests__/Combobox.spec.tsx [hunk 4]
```diff
@@ -35,159 +40,227 @@ describe('Combobox', () => {
...
         });
         describe('when blurred', () => {
             it('should render selection placeholder', () => {
-                expect(wrapper.text()).toEqual(inputPlaceholder);
+                expect(screen.findByText('type here...')).toBeDefined();
```
Comment (eszthoff) [inline src/components/SelectComponents/Combobox/__tests__/Combobox.spec.tsx:98]: I saw in the docs `toBeInDocument`. Here you use `toBeDefined`. Can you explain the difference and why you choose to use this one?

File: src/components/SelectComponents/Combobox/__tests__/Combobox.spec.tsx [hunk 5]
```diff
@@ -35,159 +40,227 @@ describe('Combobox', () => {
...
+                        noSuggestionsPlaceholder={noSuggestionsPlaceholder}
+                        onSelectionAdd={mockOnSelectionAdd}
+                        onInputValueChange={mockOnInputValueChange}
+                        onBlur={mockOnBlur}
+                        disabled
+                        selectedSuggestion={SUGGESTIONS[1]}
+                    />
+                );
+
+                expect(screen.findByText(SUGGESTIONS[1].name)).toBeDefined();
```
Comment (eszthoff) [inline src/components/SelectComponents/Combobox/__tests__/Combobox.spec.tsx:115]: this is a lot better the `querySelector` but still can be better. See https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-byrole-most-of-the-time

File: src/components/SelectComponents/Combobox/__tests__/Combobox.spec.tsx [hunk 6]
```diff
@@ -35,159 +40,227 @@ describe('Combobox', () => {
...
+                        inputPlaceholder={inputPlaceholder}
+                        noSuggestionsPlaceholder={noSuggestionsPlaceholder}
+                        onSelectionAdd={mockOnSelectionAdd}
+                        onInputValueChange={mockOnInputValueChange}
+                        onBlur={mockOnBlur}
+                        inputAttrs={{ 'data-test': true, title: 'some title' }}
+                    />
+                );
+                setFocusOnInput(viewContainer);
+                const inputField = viewContainer.querySelector('input') as Element;
```
Comment (eszthoff) [inline src/components/SelectComponents/Combobox/__tests__/Combobox.spec.tsx:133]: s/a

File: src/components/SelectComponents/Combobox/__tests__/Combobox.spec.tsx [hunk 7]
```diff
@@ -35,159 +40,227 @@ describe('Combobox', () => {
...
-                expect(wrapper.find('input').html()).toMatch('data-test="true"');
-                expect(wrapper.find('input').html()).toMatch('title="some title"');
+                expect(inputField.outerHTML).toMatch('data-test="true"');
+                expect(inputField.outerHTML).toMatch('title="some title"');
             });
             it('should render selection placeholder', () => {
-                setFocusOnInput();
-                expect(wrapper.find('input').props().placeholder).toEqual(inputPlaceholder);
+                setFocusOnInput(viewContainer);
+                const inputField = viewContainer.querySelector('input') as Element;
```
Comment (eszthoff) [inline src/components/SelectComponents/Combobox/__tests__/Combobox.spec.tsx:140]: s/a

File: src/components/SelectComponents/Combobox/__tests__/Combobox.spec.tsx [hunk 8]
```diff
@@ -35,159 +40,227 @@ describe('Combobox', () => {
...
+
+                expect(inputField.getAttribute('placeholder')).toEqual(inputPlaceholder);
             });
             it('should show the selected value as placeholder if available', () => {
-                setFocusOnInput();
-                expect(wrapper.find('input').props().placeholder).toEqual(inputPlaceholder);
+                setFocusOnInput(viewContainer);
+                const inputField = viewContainer.querySelector('input') as Element;
```
Comment (eszthoff) [inline src/components/SelectComponents/Combobox/__tests__/Combobox.spec.tsx:146]: s/a

File: src/components/SelectComponents/Combobox/__tests__/Combobox.spec.tsx [hunk 9]
```diff
@@ -35,159 +40,227 @@ describe('Combobox', () => {
...
-            wrapper.update();
-
-            expect(getInputNode()).not.toBe(document.activeElement);
-            // TODO: fixMe - the callback is not triggered, even though it does work in the UI
-            // expect(mockOnBlur).toHaveBeenCalled();
+            setFocusOnInput(viewContainer);
+            // expect(viewContainer.querySelector('input')).toBe(document.activeElement);
+            // expect(viewContainer.querySelector('.Combobox__wrapper')).toBe(document.activeElement);
         });
+        // it('should not stay focused when suggestion is selected', () => {
```
Comment (eszthoff) [inline src/components/SelectComponents/Combobox/__tests__/Combobox.spec.tsx:176]: what is the plan with these tests?

File: src/components/SelectComponents/Combobox/__tests__/Combobox.spec.tsx [hunk 10]
```diff
@@ -35,159 +40,227 @@ describe('Combobox', () => {
...
     describe('callbacks', () => {
         describe('onSelectionAdd', () => {
             it('should be called on clicking on a suggestion', () => {
-                setFocusOnInput();
+                expect(view.asFragment()).toMatchSnapshot();
+                setFocusOnInput(viewContainer);
                 expect(mockOnSelectionAdd).not.toHaveBeenCalled();
 
-                wrapper.find('li').first().children().simulate('click');
+                fireEvent.click(viewContainer.querySelector('li'));
```
Comment (eszthoff) [inline src/components/SelectComponents/Combobox/__tests__/Combobox.spec.tsx:227]: s/a for fireEvent and querySelector both

**Review by eszthoff — CHANGES_REQUESTED** [review]

File: src/components/SelectComponents/Autosuggest/__tests__/Autosuggest.spec.tsx [hunk 11]
```diff
@@ -0,0 +1,200 @@
...
+                onBlur={mockOnBlur}
+            />
+        );
+        inputNodeField = screen.getByRole('textbox');
+    });
+
+    afterEach(() => {
+        suggestionsList = [];
+        selectedSuggestions = [];
+        view.unmount();
```
[the remark this answers is not in the document]
Comment (PavelKirvelTextkernel) [reply src/components/SelectComponents/Autosuggest/__tests__/Autosuggest.spec.tsx:49]: I think we can remove it

File: src/components/SelectComponents/Autosuggest/__tests__/Autosuggest.spec.tsx [hunk 12]
```diff
@@ -0,0 +1,200 @@
...
+                suggestions={suggestionsList}
+                suggestionToString={suggestionToString}
+                inputPlaceholder={inputPlaceholder}
+                onSelectionAdd={mockOnSelectionAdd}
+                onSelectionRemove={mockOnSelectionRemove}
+                onInputValueChange={mockOnInputValueChange}
+                numberOfVisibleTags={numberOfVisibleTags}
+                onBlur={mockOnBlur}
+            />
+        );
```
[the remark this answers is not in the document]
Comment (PavelKirvelTextkernel) [reply src/components/SelectComponents/Autosuggest/__tests__/Autosuggest.spec.tsx:42]: Yes it's a good idea

**Review by eszthoff — CHANGES_REQUESTED** [review]

File: src/components/SelectComponents/Combobox/__tests__/Combobox.spec.tsx [hunk 13]
```diff
@@ -17,178 +19,209 @@ describe('Combobox', () => {
...
+                const newProps = {
+                    suggestions,
+                    suggestionToString,
+                    inputPlaceholder,
+                    noSuggestionsPlaceholder,
+                    onSelectionAdd: mockOnSelectionAdd,
+                    onInputValueChange: mockOnInputValueChange,
+                    onBlur: mockOnBlur,
+                    selectedSuggestion: SUGGESTIONS[1],
+                };
```
Comment (eszthoff) [inline src/components/SelectComponents/Combobox/__tests__/Combobox.spec.tsx:129]: are all these props new and needed to be listed here?

File: src/components/SelectComponents/Combobox/__tests__/Combobox.spec.tsx [hunk 14]
```diff
@@ -17,178 +19,209 @@ describe('Combobox', () => {
...
+            await setFocus();
+            expect(getInputNode(viewContainer)).toBe(document.activeElement);
 
-            wrapper.find('input').simulate('keyDown', { key: 'Escape' });
-            wrapper.update();
+            await userEvent.keyboard('[Escape]');
 
-            expect(getInputNode()).not.toBe(document.activeElement);
+            expect(getInputNode(viewContainer)).not.toBe(document.activeElement);
             // TODO: fixMe - the callback is not triggered, even though it does work in the UI
```
Comment (eszthoff) [inline src/components/SelectComponents/Combobox/__tests__/Combobox.spec.tsx:163]: please delete this TODO if it is working now as expected. Thanks

File: src/components/SelectComponents/Combobox/__tests__/Combobox.spec.tsx [hunk 15]
```diff
@@ -17,178 +19,209 @@ describe('Combobox', () => {
...
+        //         // expect(mockOnBlur).toHaveBeenCalled();
+        //         done();
+        //     });
+        // });
+        it('should blur on pressing ENTER button', async () => {
+            await setFocus();
+            await userEvent.keyboard('[Enter]');
+
+            expect(getInputNode(viewContainer)).not.toBe(document.activeElement);
             // TODO: fixMe - the callback is not triggered, even though it does work in the UI
```
Comment (eszthoff) [inline src/components/SelectComponents/Combobox/__tests__/Combobox.spec.tsx:182]: please delete this comment

File: src/components/SelectComponents/Combobox/__tests__/Combobox.spec.tsx [hunk 16]
```diff
@@ -17,178 +19,209 @@ describe('Combobox', () => {
...
         describe('onSelectionAdd', () => {
-            it('should be called on clicking on a suggestion', () => {
-                setFocusOnInput();
+            it('should be called on clicking on a suggestion', async () => {
+                expect(view.asFragment()).toMatchSnapshot();
+                await setFocus();
                 expect(mockOnSelectionAdd).not.toHaveBeenCalled();
 
-                wrapper.find('li').first().children().simulate('click');
+                await userEvent.click(viewContainer.querySelector('li'));
```
Comment (eszthoff) [inline src/components/SelectComponents/Combobox/__tests__/Combobox.spec.tsx:193]: please select by role, or add a comment to fix it in the next ticket.

File: src/components/SelectComponents/ComboboxMulti/ComboboxMulti.tsx [hunk 17]
```diff
@@ -76,6 +76,7 @@ export function ComboboxMulti<S>({
                 <IoMdArrowDropup
                     {...elem('dropdownIcon', { disabled })}
                     {...getToggleButtonProps({ disabled, onClick: blur })}
+                    role="img"
```
Comment (eszthoff) [inline src/components/SelectComponents/ComboboxMulti/ComboboxMulti.tsx:79]: I think this should have `role="button"`

File: src/components/SelectComponents/ComboboxMulti/ComboboxMulti.tsx [hunk 18]
```diff
@@ -104,6 +105,7 @@ export function ComboboxMulti<S>({
                         e?.stopPropagation();
                     },
                 })}
+                role="img"
```
Comment (eszthoff) [inline src/components/SelectComponents/ComboboxMulti/ComboboxMulti.tsx:108]: I think this should have `role="button"`

File: src/components/SelectComponents/ComboboxMulti/__tests__/ComboboxMulti.spec.tsx [hunk 19]
```diff
@@ -0,0 +1,108 @@
...
+        it('should render noSuggestions placeholder when empty suggestions list is passed', async () => {
+            const newProps = {
+                suggestions: [],
+                suggestionToString,
+                inputPlaceholder,
+                noSuggestionsPlaceholder,
+                onSelectionAdd: mockOnSelectionAdd,
+                onInputValueChange: mockOnInputValueChange,
+                onBlur: mockOnBlur,
+            };
```
Comment (eszthoff) [inline src/components/SelectComponents/ComboboxMulti/__tests__/ComboboxMulti.spec.tsx:93]: do we need to define all of these props here?

File: src/components/SelectComponents/SelectBase/__tests__/SelectBase.spec.tsx [hunk 20]
```diff
@@ -20,156 +22,196 @@ describe('SelectBase', () => {
...
+                listRenderer: (listProps) => <SuggestionsList {...listProps} />,
+                focusedRenderer: mockRender,
+                blurredRenderer: mockRender,
+                onSelectionAdd: mockOnSelectionAdd,
+                onInputValueChange: mockOnInputValueChange,
+                onClearAllSelected: mockOnClearAllSelected,
+                onBlur: mockOnBlur,
+                highlightOnEmptyInput: true,
+                showArrow: true,
+            };
```
Comment (eszthoff) [inline src/components/SelectComponents/SelectBase/__tests__/SelectBase.spec.tsx:81]: maybe we don't need all of these props

File: src/components/SelectComponents/SuggestionsList/SuggestionsList.tsx [hunk 21]
```diff
@@ -93,7 +93,7 @@ export function SuggestionsList<S>({
                 return (
                     // eslint-disable-next-line react/no-array-index-key
                     <ListItem key={i}>
-                        <div {...elem('loaderItem')}>
+                        <div {...elem('loaderItem')} role="group">
```
Comment (eszthoff) [inline src/components/SelectComponents/SuggestionsList/SuggestionsList.tsx:96]: why did you decide to use this role here?

**Review by eszthoff — CHANGES_REQUESTED** [review]

File: src/components/SelectComponents/Autosuggest/__tests__/Autosuggest.spec.tsx [hunk 22]
```diff
@@ -97,103 +87,122 @@ describe('Autosuggest', () => {
...
+                numberOfVisibleTags,
+                onBlur: mockOnBlur,
+                inputAttrs: { 'data-test': true, title: 'some title' },
+            };
+            rerenderView(newProps);
+            await setFocus();
             await userEvent.type(inputNodeField, 'driver');
-            expect(view.container.querySelectorAll('.SuggestionsList__loaderItem')).toHaveLength(5);
+
+            expect(screen.getAllByRole('presentation')).toHaveLength(5);
```
[the remark this answers is not in the document]
Comment (PavelKirvelTextkernel) [reply src/components/SelectComponents/Autosuggest/__tests__/Autosuggest.spec.tsx:122]: :+1:

File: src/components/SelectComponents/Autosuggest/__tests__/Autosuggest.spec.tsx [hunk 23]
```diff
@@ -0,0 +1,157 @@
...
+            const newProps = {
+                isLoading: false,
+                suggestions: suggestionsList,
+                inputAttrs: { 'data-test': true, title: 'some title' },
+            };
+            rerenderView(newProps);
+            await setFocus();
+
+            expect(view.asFragment()).toMatchSnapshot();
+            expect(screen.getAllByRole('presentation')).toHaveLength(8);
```
[the remark this answers is not in the document]
Comment (PavelKirvelTextkernel) [reply src/components/SelectComponents/Autosuggest/__tests__/Autosuggest.spec.tsx:97]: :+1:

File: src/components/SelectComponents/Autosuggest/__tests__/Autosuggest.spec.tsx [hunk 24]
```diff
@@ -0,0 +1,146 @@
...
+                isLoading: true,
+                suggestions: suggestionsList,
+                allowMixingSuggestionsAndLoading: true,
+            };
+            rerenderView(newProps);
+            await setFocus();
+            await userEvent.type(inputNodeField, 'driver');
+
+            expect(screen.getAllByRole('presentation')).toHaveLength(7);
+            expect(screen.getAllByRole('group')).toHaveLength(5);
```
[the remark this answers is not in the document]
Comment (PavelKirvelTextkernel) [reply src/components/SelectComponents/Autosuggest/__tests__/Autosuggest.spec.tsx:137]: :+1:

File: src/components/SelectComponents/Autosuggest/__tests__/Autosuggest.spec.tsx [hunk 25]
```diff
@@ -0,0 +1,146 @@
...
+        });
+        it('should render empty component correctly when focused', async () => {
+            expect(view.asFragment()).toMatchSnapshot();
```
[the remark this answers is not in the document]
Comment (PavelKirvelTextkernel) [reply src/components/SelectComponents/Autosuggest/__tests__/Autosuggest.spec.tsx:140]: :+1:

File: src/components/SelectComponents/Combobox/__tests__/Combobox.spec.tsx (hunk 13 above)
Comment (PavelKirvelTextkernel) [reply src/components/SelectComponents/Combobox/__tests__/Combobox.spec.tsx:129]: :+1:

File: src/components/SelectComponents/Combobox/__tests__/Combobox.spec.tsx (hunk 14 above)
Comment (PavelKirvelTextkernel) [reply src/components/SelectComponents/Combobox/__tests__/Combobox.spec.tsx:163]: :+1:

File: src/components/SelectComponents/Combobox/__tests__/Combobox.spec.tsx (hunk 15 above)
Comment (PavelKirvelTextkernel) [reply src/components/SelectComponents/Combobox/__tests__/Combobox.spec.tsx:182]: :+1:

File: src/components/SelectComponents/Combobox/__tests__/Combobox.spec.tsx (hunk 16 above)
Comment (PavelKirvelTextkernel) [reply src/components/SelectComponents/Combobox/__tests__/Combobox.spec.tsx:193]: :+1:

File: src/components/FieldWrapper/FieldWrapper.tsx [hunk 26]
```diff
@@ -88,13 +88,15 @@ export const FieldWrapper = React.forwardRef<HTMLDivElement, Props>(
                             {...elem('dropdownIcon')}
                             onClick={handleArrowClick}
                             onKeyDown={handleArrowKeyDown}
+                            role="img"
```
[the remark this answers is not in the document]
Comment (PavelKirvelTextkernel) [reply src/components/FieldWrapper/FieldWrapper.tsx:91]: :+1:

File: src/components/FieldWrapper/FieldWrapper.tsx [hunk 27]
```diff
@@ -88,13 +88,15 @@ export const FieldWrapper = React.forwardRef<HTMLDivElement, Props>(
...
                         />
                     ) : (
                         <IoMdArrowDropdown
                             tabIndex={0}
                             {...elem('dropdownIcon')}
                             onClick={handleArrowClick}
                             onKeyDown={handleArrowKeyDown}
+                            role="img"
```
[the remark this answers is not in the document]
Comment (PavelKirvelTextkernel) [reply src/components/FieldWrapper/FieldWrapper.tsx:99]: :+1:

File: src/components/SelectComponents/Autosuggest/__tests__/Autosuggest.spec.tsx [hunk 28]
```diff
@@ -1,6 +1,8 @@
 import React from 'react';
 import { render, screen } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
+import '@testing-library/jest-dom';
+import { create } from 'react-test-renderer';
```
[the remark this answers is not in the document]
Comment (PavelKirvelTextkernel) [reply src/components/SelectComponents/Autosuggest/__tests__/Autosuggest.spec.tsx]: :+1:

File: src/components/SelectComponents/Combobox/__tests__/Combobox.spec.tsx [hunk 29]
```diff
@@ -17,178 +19,209 @@ describe('Combobox', () => {
...
+            expect(screen.getAllByRole('presentation')).toHaveLength(1);
+            expect(screen.getByText(noSuggestionsPlaceholder)).toBeInTheDocument();
         });
-        it('should render all suggestions from the list', () => {
-            setFocusOnInput();
+        it('should render all suggestions from the list', async () => {
+            await setFocus();
 
-            expect(wrapper.find('li')).toHaveLength(suggestions.length);
+            expect(screen.getAllByRole('presentation')).toHaveLength(suggestions.length);
```
[the remark this answers is not in the document]
Comment (PavelKirvelTextkernel) [reply src/components/SelectComponents/Combobox/__tests__/Combobox.spec.tsx:78]: :+1:

File: src/components/SelectComponents/ComboboxMulti/ComboboxMulti.tsx (hunk 17 above)
Comment (PavelKirvelTextkernel) [reply src/components/SelectComponents/ComboboxMulti/ComboboxMulti.tsx:79]: :+1:

File: src/components/SelectComponents/SuggestionsList/SuggestionsList.tsx (hunk 21 above)
Comment (PavelKirvelTextkernel) [reply src/components/SelectComponents/SuggestionsList/SuggestionsList.tsx:96]: I've replaced by listitem, i use it in Autosuggest.spec.tsx

File: src/components/SelectComponents/Autosuggest/__tests__/Autosuggest.spec.tsx [hunk 30]
```diff
@@ -0,0 +1,157 @@
...
+
+            expect(view.asFragment()).toMatchSnapshot();
+            expect(screen.getAllByRole('presentation')).toHaveLength(8);
+        });
+        it('should render component with suggestions', async () => {
+            suggestionsList = SUGGESTIONS.slice(1, 20);
+            const newProps = {
+                isLoading: false,
+                suggestions: suggestionsList,
+                inputAttrs: { 'data-test': true, title: 'some title' },
```
[the remark this answers is not in the document]
Comment (PavelKirvelTextkernel) [reply src/components/SelectComponents/Autosuggest/__tests__/Autosuggest.spec.tsx:104]: :+1:

File: src/components/SelectComponents/Autosuggest/__tests__/Autosuggest.spec.tsx [hunk 31]
```diff
@@ -0,0 +1,157 @@
...
+            };
+            rerenderView(newProps);
+            await setFocus();
+            await userEvent.type(inputNodeField, 'driver');
+
+            expect(view.asFragment()).toMatchSnapshot();
```
[the remark this answers is not in the document]
Comment (PavelKirvelTextkernel) [reply src/components/SelectComponents/Autosuggest/__tests__/Autosuggest.spec.tsx:112]: :+1:

File: src/components/SelectComponents/ComboboxMulti/ComboboxMulti.tsx (hunk 18 above)
Comment (PavelKirvelTextkernel) [reply src/components/SelectComponents/ComboboxMulti/ComboboxMulti.tsx]: :+1:

File: src/components/SelectComponents/Autosuggest/__tests__/Autosuggest.spec.tsx [hunk 32]
```diff
@@ -0,0 +1,157 @@
...
+
+            expect(inputField.outerHTML).toMatch('data-test="true"');
+            expect(inputField.outerHTML).toMatch('title="some title"');
+        });
+        it('should initially render focused component with suggestions list correctly', async () => {
+            suggestionsList = SUGGESTIONS.slice(0, 8);
+            const newProps = {
+                isLoading: false,
+                suggestions: suggestionsList,
+                inputAttrs: { 'data-test': true, title: 'some title' },
```
[the remark this answers is not in the document]
Comment (PavelKirvelTextkernel) [reply src/components/SelectComponents/Autosuggest/__tests__/Autosuggest.spec.tsx:92]: :+1:

File: src/components/SelectComponents/Combobox/__tests__/Combobox.spec.tsx (hunk 13 above)
Comment (PavelKirvelTextkernel) [reply src/components/SelectComponents/Combobox/__tests__/Combobox.spec.tsx:131]: Fixed

**Review by eszthoff — APPROVED** [review]

**Pull request merged**